CHANGELOG.md

```diff
@@ -1 +1,164 @@
+- Fix InhibitIPv4 nil panic [moby/moby#40596](https://github.com/moby/moby/pull/40596)
+- Fix VFP leak in Windows overlay network deletion [moby/moby#40596](https://github.com/moby/moby/pull/40596) [moby/libnetwork#2524](https://github.com/moby/libnetwork/pull/2524)
+
+### Packaging
+
+* docker.service: Add multi-user.target to After= in unit file [moby/moby#41297](https://github.com/moby/moby/pull/41297)
+* docker.service: Allow socket activation [moby/moby#37470](https://github.com/moby/moby/pull/37470)
+
+### Rootless
+
+* rootless: graduate from experimental [moby/moby#40759](https://github.com/moby/moby/pull/40759)
++ Add dockerd-rootless-setuptool.sh [moby/moby#40950](https://github.com/moby/moby/pull/40950)
+* Support `--exec-opt native.cgroupdriver=systemd` [moby/moby#40486](https://github.com/moby/moby/pull/40486)
+
+### Security
+
+* Fix CVE-2019-14271 loading of nsswitch based config inside chroot under Glibc [moby/moby#39612](https://github.com/moby/moby/pull/39612)
+* seccomp: Whitelist `clock_adjtime`. `CAP_SYS_TIME` is still required for time adjustment [moby/moby#40929](https://github.com/moby/moby/pull/40929)
+* seccomp: Add openat2 and faccessat2 to default seccomp profile [moby/moby#41353](https://github.com/moby/moby/pull/41353)
+* seccomp: allow 'rseq' syscall in default seccomp profile [moby/moby#41158](https://github.com/moby/moby/pull/41158)
+* seccomp: Remove dependency in dockerd on libseccomp [moby/moby#41395](https://github.com/moby/moby/pull/41395)
+* seccomp: allow syscall membarrier [moby/moby#40731](https://github.com/moby/moby/pull/40731)
+* seccomp: whitelist io-uring related system calls [moby/moby#39415](https://github.com/moby/moby/pull/39415)
+* Add default sysctls to allow ping sockets and privileged ports with no capabilities [moby/moby#41030](https://github.com/moby/moby/pull/41030)
+- Fix seccomp profile for clone syscall [moby/moby#39308](https://github.com/moby/moby/pull/39308)
+
+### Swarm
+
++ Add support for swarm jobs [moby/moby#40307](https://github.com/moby/moby/pull/40307)
++ Add support for sending down service Running and Desired task counts [moby/moby#39231](https://github.com/moby/moby/pull/39231)
++ service: support --mount type=bind,bind-nonrecursive [moby/moby#38788](https://github.com/moby/moby/pull/38788)
++ Support ulimits on Swarm services. [moby/moby#41284](https://github.com/moby/moby/pull/41284) [docker/cli#2712](https://github.com/docker/cli/pull/2712)
++ Add support for capabilities options in services [moby/moby#39173](https://github.com/moby/moby/pull/39173)
++ Service cap-add/cap-drop: add special "RESET" value [docker/cli#2709](https://github.com/docker/cli/pull/2709)
+* Replace swarm Capabilites API with cap add/drop API [moby/moby#41249](https://github.com/moby/moby/pull/41249)
```

Comment:
I don't think this should be in the changelog, since the capabilities API was never released.

Reply:
Really all these capabilities things can be merged into 1.